stable/unifi/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+appVersion: 5.8.24
+description: Ubiqiti Network's Unifi Controller
+name: unifi
+version: 0.1.13
```

Comment:
Why exactly this version?

Reply:
As I was iterating through local development of this chart, I would bump the minor version each time.  I wasn't sure what a proper fresh version should be (0.0.0 or 1.0.0 ?) - any guidance?

Reply:
pegging to 0.1.0 for now until the chart is initially released